src/main/java/by/exadel/internship/service/impl/InternshipServiceImpl.java

```diff
@@ -131,6 +132,21 @@ public UserInternshipDTO saveInternship(UserInternshipDTO internshipDTO) {
         return userInternshipDTO;
     }
 
+    @Transactional
+    public void addUser(UUID userId, UUID internshipId) {
+        userRepository.findById(userId)
+                .orElseThrow(() -> new NotFoundException("User Not Found"));
+
+        internshipRepository.findById(internshipId)
+                .orElseThrow(() -> new NotFoundException("Internship Not Found"));
+
+        int exist = internshipRepository.checkUserExists(userId, internshipId);
+        if (exist > 0) {
+            throw new RuntimeException("ALREADY EXIST");
```

Comment:
what `ALREADY EXIST`?) May be, `User is already assigning to Intership`.

Reply:
done